NSS-Day-Cohort-49/react-nutshell-state-slingers
# Mat events

#### Changes Made
1.EventProvider, EventForm, EventList, EventCard
1.made changes to application views
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout branchname
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > When a user navigates to the events page, their events should show as well as their friends. their events should be in a light red/pink, their friends in cornsilk and italic. The soonest event should be a more bold style.
> When a user clicks to add an event, an event form should open. After entering all fields, an event can be saved (or canceled)
>A user may edit their own events, showing prepopulated data
>A user may view current weather conditions by clicking the show weather button
> A user should be able to view weather conditions for days fewer than 7 days away by clicking the corresponding weather button on the event.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
